bigchaindb/tep/transactional_election.py

```diff
@@ -0,0 +1,38 @@
+
+class TransactionalElection(object):
+
+    version = '0.0'
+    name = 'base-class'
+
+    def __init__(self):
+        pass
+
+    def propose(self, election_object):
+        """Initiate a new election"""
+
+        raise NotImplementedError
+
+    def list(self, election_id):
+        """List the election with id `election_id`"""
+
+        raise NotImplementedError
+
+    def is_valid_proposal(self, election_object):
```

Comment:
If I understood it correctly, `is_valid_proposal` checks if the node wants to vote for the given proposal. Even if `is_valid_proposal` returns `False`, the proposal can be still voted for by the majority and the validator will be added. 

What do you think about renaming it to better reflect this? I can think of e. g. `agree_on_proposal`. The current name makes me think that the function checks for the structural validity of the proposal, like general transaction validity.

Reply:
True, I misunderstood it at first, sorry.